pages/caritonaranjo/index.html

```diff
@@ -0,0 +1,93 @@
+              </p>
+              <button class="button__primary">MORE</button>
+            </div>
+        </div>
+
+        <div class="card__block">
+          <div class="card__image">
+            <img class="card__img--top" src="images/romance.jpeg" alt="Card image" />
+          </div>
+            <div class="card__body">
+              <h4 class="card__title">ROMANCE</h4>
+              <p class="card__text">
+                According to the Romance Writers of America, "Two basic elements comprise every romance novel: a central love story and an emotionally-satisfying and optimistic ending.
+                " Both the conflict and the climax of the novel should be directly related to that core theme of developing a romantic relationship, although the novel can also contain subplots that do not specifically relate to the main characters' romantic love.
+              </p>
+              <button class="button__primary">MORE</button>
+            </div>
+        </div>
+
+        <div class="card__block">
+          <div class="card__image">
+            <img class="card__img--top" src="images/fantasy.jpg" alt="Card image" />
+          </div>
+            <div class="card__body">
+              <h4 class="card__title">FANTASY</h4>
+              <p class="card__text">
+                Fantasy is a genre that uses magic and other supernatural forms as a primary element of plot, theme, and/or setting.
+                Fantasy is generally distinguished from science fiction and horror by the expectation that it steers clear of technological and macabre themes, respectively, though there is a great deal of overlap between the three (collectively known as speculative fiction or science fiction/fantasy)
+
+              </p>
+              <button class="button__primary">MORE</button>
+            </div>
+        </div>
+        <!--Controls-->
+        <section class="controls__side">
+          <button class="btn__side--right" href="#carousel" data-slide="prev"> 
+          <span class="btn__img "><</span>
+        </button>
+          <button class=" btn__side--left" href="#carousel" data-slide="next">
+          <span class="btn__img">></span>
+          </button>
+        </section>
+        <!--/.Controls-->
+      </div>
+      <!--Indicators-->
+      <section >
+        <ul class="carousel__indicators">
+        <li class="carousel__indicators--list "><button class=" active carousel__indicators--dot" data-target="#carousel"
+          data-slide-to="0">
+        </button>
+        </li>
+        <li class="carousel__indicators--list"><button  class="carousel__indicators--dot" data-target="#carousel" data-slide-to="1"></button></li>
+        <li class="carousel__indicators--list"><button  class="carousel__indicators--dot" data-target="#carousel" data-slide-to="2"></button></li>
+        <li class="carousel__indicators--list"><button class="carousel__indicators--dot" data-target="#carousel" data-slide-to="3"></button></li>
```

Comment:
This line is too long, could you break it?

```suggestion
<li class="carousel__indicators--list">
   <button class="carousel__indicators--dot" data-target="#carousel" data-slide-to="3"></button>
</li>
```